src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const titleCheck = movie.title.toLowerCase();
+    const descriptionCheck = movie.description.toLowerCase();
+
+    const queryCheck = query.trim().toLowerCase();
```

Comment:
It would be nice to delete unnecessary spaces between words in the search query. If I search 'love    actually' (with two spaces) I will not get results. You can use something like this:
`const queryCheck = query.toLowerCase().split(' ').filter(Boolean).join(' ');`